app/src/main/java/nl/tudelft/b_b_w/model/GetDatabaseHandler.java

```diff
@@ -66,7 +63,7 @@ public final String getContactName(String hash) {
 
         // When returning an exception the whole program crashes,
         // but we want to preserve the state.
-        if (cursor.getCount() < 1) return null;
+        if (cursor.getCount() < 1) return "Unknown";
```

Comment:
This method will be removed as soon as your changes are merged. This method then becomes redundant because you also wrote the same thing.